packages/integration-tests/package.json

```diff
@@ -12,7 +12,7 @@
   "publishConfig": {
     "access": "public"
   },
-  "homepage": "https://github.com/nasa/cumulus/tree/master/packages/integration-tests",
+  "homepage": "https://github.com/nasa/cumulus/tree/master/packages/integration-tests/README.md",
```

Comment:
As far as I know, linking to a folder root on Github will display the README for that directory